src/Yield.Infrastructure/Repositories/Interfaces/IPlotRepository.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Yield.Core.Entities;
+
+namespace Yield.Infrastructure.Repositories.Interfaces
+{
+    public interface IPlotRepository
+    {
+        Task<IEnumerable<Plot>> GetPlots(Guid AllotmentId);
+        Task<Plot> GetPlot(Guid AllotmentId, Guid Id);
```

Comment:
Lowecase params.
```suggestion
        Task<Plot> GetPlot(Guid allotmentId, Guid id);
```

Reply:
Whoops, thanks for checking this for me @SvenPam 😅